docs/persist-state.md

```diff
@@ -0,0 +1,18 @@
+# Persistence
+In Android, there are two major cases in which persistence must be considered:
+
+#### Configuration Changes / Back Stack
+Traditionally, you would use savedInstanceState to save important information. However, with Mavericks, you will get the exact same instance of your ViewModel back so you have everything you need to re-render the view. Generally, screens won't need to implement savedInstanceState with Mavericks.
```

Comment:
Maybe worth mentioning this is standard Jetpack ViewModel behavior, and link to jetpack docs if there are related ones